packages/components/MenuButton/src/MacOSMenuButtonNativeComponent.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ * @format
+ */
+
+import type { HostComponent, ImageSourcePropType, ViewProps } from 'react-native';
+
+import type { BubblingEventHandler, Int32 } from 'react-native/Libraries/Types/CodegenTypes';
+import codegenNativeComponent from 'react-native/Libraries/Utilities/codegenNativeComponent';
+
+interface MenuItem {
+  title?: string;
+  image?: ImageSourcePropType;
+  enabled?: boolean;
+  tooltip?: string;
+  identifier?: string;
+  hasSubmenu?: boolean;
+  submenu: MenuItem[];
+}
+
+export interface NativeProps extends ViewProps {
+  content?: string;
+  image?: ImageSourcePropType;
+  enabled?: boolean;
+  menu?: MenuItem;
```

Comment:
tooltip missing?

Reply:
Looks like its toolTip in native:
https://github.com/microsoft/fluentui-react-native/blob/ced28105fa91b14cfe655651cc39799354adfba8/packages/components/MenuButton/macos/FRNMenuButtonManager.m#L51

but tooltip in JS:
https://github.com/microsoft/fluentui-react-native/blob/ced28105fa91b14cfe655651cc39799354adfba8/packages/components/MenuButton/src/MenuButton.macos.tsx#L111

Are properties not case sensitive? -- Is this a bug?

Reply:
looks like its tooltip - inherited from view.  -- So it shouldn't actually be part of this component spec.

Reply:
I can't remember if the case sensitivity has bit us before. If we fix this on the native side with `RCT_REMAP_PROPERTY` or something, I'd be happy to do so.